src/pages/ProductsList/ProductsListComponents/RightFilter.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Card from '../../../components/ProductCard/ProductCard';
+const RightFilter = () => {
+  const [list, setList] = useState([]);
+  console.log(list);
+
+  useEffect(() => {
+    fetch('data/ProductCard.json')
+      .then(res => res.json())
+      .then(result => setList(result));
+  }, []);
+
+  return (
+    <Right>
+      <Sorting>
+        <Popular>인기순✔️</Popular>
+      </Sorting>
+      <Products>
+        {list.length > 0 &&
+          list.map(comment => {
+            return (
+              <Card
+                key={comment.id}
+                product_image={comment.product_image[0].image}
+                author={comment.author}
+                name={comment.name}
+                price={comment.price}
+              />
+            );
+          })}
```

Comment:
```suggestion
        {list.length > 0 &&
          list.map(({id, author, name, price, product_image}) => {
            return (
              <Card
                key={comment.id}
                product_image={comment.product_image[0].image}
                author={comment.author}
                name={comment.name}
                price={comment.price}
              />
            );
          })}
```
매개변수 자리에서 바로 구조분해할당해서 사용하면 좀 더 깔끔해지겠네요!

Reply:
수정하였습니다.